arches/management/commands/add_users.py

```diff
@@ -0,0 +1,189 @@
+            self.load_users_from_csv(options)
+
+    def add_test_users(self, options):
+        user_count = options.get("user_count")
+
+        profiles = []
+        for current in range(1, user_count + 1):
+            profiles.append(
+                {
+                    "name": f"tester{current}",
+                    "email": f"tester{current}@test.com",
+                    "password": "Test12345!",
+                    "groups": ["Graph Editor", "Resource Editor"],
+                }
+            )
+
+        if options.get("power_user"):
+            profiles = list(
+                profiles
+                + [
+                    {
+                        "name": "dev",
+                        "email": "dev@test.com",
+                        "password": "dev",
+                        "groups": [
+                            "Graph Editor",
+                            "Resource Editor",
+                            "Resource Exporter",
+                            "Resource Reviewer",
+                            "Application Administrator",
+                            "Crowdsource Editor",
+                            "Guest",
+                            "RDM Administrator",
+                            "Resource Reviewer",
+                            "System Administrator",
+                            "Developer",
+                        ],
+                    }
+                ]
+            )
+
+            try:
+                dev_group = Group.objects.create(name="Developer")
+                dev_perms = Permission.objects.all().values("id")
+                perm_ids = [int(perm["id"]) for perm in dev_perms]
+                for permission in perm_ids:
+                    dev_group.permissions.add(permission)
+            except Exception as e:
+
+                print(e)
+
+        plugins = models.Plugin.objects.all()
+        etl_modules = models.ETLModule.objects.all()
+        for profile in profiles:
+            try:
+                user = User.objects.create_user(
+                    username=profile["name"],
+                    email=profile["email"],
+                    password=profile["password"],
+                )
+                if user.username == "dev":
+                    user.is_staff = True
+                    user.first_name = "Dev"
+                    user.last_name = "User"
+                    for plugin in plugins:
+                        assign_perm("change_plugin", user, plugin)
+                        assign_perm("add_plugin", user, plugin)
+                        assign_perm("delete_plugin", user, plugin)
+                        assign_perm("view_plugin", user, plugin)
+                    for etl_module in etl_modules:
+                        assign_perm("view_etlmodule", user, etl_module)
+                user.save()
+                print(
+                    f"Added test user: {user.username}, password: {profile['password']}"
+                )
```

Comment:
Can we update the print statements throughout to `self.stdout.write()` while here? These print statements are noisy when run during the arches-querysets test suite.

Reply:
good to see you still lurking!  😃  Done.